.github/workflows/Build-and-deploy-nbs6-win-container.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build NBS6 Windows Container Build
+on:
+  push:
+    branches:
+      - docker-nbs6-win
+    # paths:
+    #   - "cdc-sandbox/nbs6-win-container/**"
+jobs:
+  push-call-build-windows-container-workflow:
+    name: On Push Build Windows Container
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Build-windows-container.yaml@docker-nbs6-win
```

Comment:
Need to change the reference from a branch to target version (or main)

Reply:
Updated to main